leader-election-impl/src/main/java/com/palantir/leader/PaxosLeaderElectionService.java

```diff
@@ -141,6 +145,8 @@ public LeadershipToken blockOnBecomingLeader() throws InterruptedException {
                     return new PaxosLeadershipToken(greatestLearned);
                 } else if (leadingStatus == StillLeadingStatus.NO_QUORUM) {
                     // If we don't have quorum we should just retry our calls.
+                    long backoffTime = (long) (noQuorumMaxDelay * Math.random());
+                    log.debug("Waiting for [{}] ms before rerequesting leadership status", SafeArg.of("waitTimeMs", backoffTime));
```

Comment:
Where does it actually backoff?

Reply:
... whoops. 

Sorry, that's a pretty big slip. Fixed, thanks for spotting >_>